contrib/dist/linux/buildrpm.sh

```diff
@@ -80,6 +180,14 @@ unset first
 echo "--> Found Open MPI version: $version"
 
 #
+# Try to unpack spec file from tarball
+#
+
+if test "$unpack_spec" = "1"; then
```

Comment:
Super minor nit, but something I try to propagate to shell coders... 😄 

When using numerical values in shell variables:
* You can assign the value without quotes: `foo=1`
* You can test the value without quotes (i.e., without promoting the integer to a string) and use `test` operators such as `-eq`, `-ne`, etc.  E.g.: `test $foo -eq 1`